Changelog.md

```diff
@@ -247,7 +247,7 @@ concurrently using this.
 
 * Add the following to `Streamly.Prelude`:
     * `unfold`, `fold`, `scan` and `postscan`
-    * `concatUnfold` to concat a stream after unfolding each element
+    * `unfoldMany` to concat a stream after unfolding each element
```

Comment:
Revert this change. We need to add a deprecation note in the unreleased section at the top.